### DISCUSSION TITLE
驾照考试记录 - WxW

### DISCUSSION THREAD

**Discussion by giscus** [Whbbit1999/wxw-blog-comments#5] [General]
# 驾照考试记录 - WxW

要考驾驶证了，但是我好像什么都不懂。用这篇文章记录一下学习驾照的经历。

https://www.whbbit.cn/2023/02/08/%E9%A9%BE%E7%85%A7%E8%80%83%E8%AF%95%E8%AE%B0%E5%BD%95/

**Comment by Whbbit1999** [comment]
2022/02/22，科一过了，98

**Reply by Whbbit1999** [reply]
牛哇

**Comment by Whbbit1999** [comment]
2022/03/11，科二也过了😁

**Reply by Whbbit1999** [reply]
哈哈哈，祝你也顺利通过考试😁

**Comment by Whbbit1999** [comment]
科三也过了，今天考科四去(｢･ω･)｢嘿

**Reply by Whbbit1999** [reply]
科四也过了(｢･ω･)｢嘿
